### PR TITLE
[NUI][TCSACR-349] Add and deprecate Button EventArgs classes

### PR DESCRIPTION
**Pull request by jaehyun0cho**
Add ClickedEventArgs class.
Deprecate Button.ClickEventArgs class.

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Comment by jaehyun0cho** [comment]
I don't have authority to set Labels to "Review/Requested".
Please give me authority to set Labels.
Thank you.

File: docs/application/dotnet/guides/nui/nui-components/Button.md [hunk 1]
```diff
@@ -136,18 +136,18 @@ Following output is generated when the Button is created using the defined style
 
 ![ButtonProperty](./media/ButtonProperty.PNG)
 
-## Responding to ClickEvent
+## Responding to Clicked
 
 When you click a Button, the Button instance receives a click event.
 You can declare the click event handler as follows:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/nui-components/Button.md:142]: Suggestion. How about changing "click" to "clicked"?

```suggestion
When you click a Button, the Button instance receives a clicked event.
You can declare the clicked event handler as follows:
```

File: docs/application/dotnet/guides/nui/nui-components/CheckBox.md [hunk 2]
```diff
@@ -41,17 +41,17 @@ Following output is generated when a checkbox is created using style:
 ![CheckBox2](./media/CheckBox2.png)
 
 
-## Responding to ClickEvent
+## Responding to Clicked
 When you click a CheckBox, the CheckBox instance receives a click event.
 You can declare the click event handler as follows:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/nui-components/CheckBox.md:46]: Suggestion. How about changing "click" to "clicked"?

```suggestion
When you click a CheckBox, the CheckBox instance receives a clicked event.
You can declare the clicked event handler as follows:
```

File: docs/application/dotnet/guides/nui/nui-components/RadioButton.md [hunk 3]
```diff
@@ -41,17 +41,17 @@ Following output is generated when a radio button is created using style:
 ![RadioButton2](./media/RadioButton2.png)
 
 
-## Responding to ClickEvent
+## Responding to Clicked
 When you click a radio button, the radio button instance receives a click event.
 You can declare the click event handler as follows:
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/nui/nui-components/RadioButton.md:46]: Suggestion. How about changing "click" to "clicked"?

```suggestion
When you click a radio button, the radio button instance receives a clicked event.
You can declare the clicked event handler as follows:
```

**Review by mijin85cho — COMMENTED** [review]
Hello, @jaehyun0cho

With this PR, I have below suggestions for minor update.
Would you please check these and let me know your opinion?

File: docs/application/dotnet/guides/nui/nui-components/Button.md (hunk 1 above)
Comment (jaehyun0cho) [reply docs/application/dotnet/guides/nui/nui-components/Button.md]: Thank you for your suggestion.
I have updated the patch as you suggested.

File: docs/application/dotnet/guides/nui/nui-components/CheckBox.md (hunk 2 above)
Comment (jaehyun0cho) [reply docs/application/dotnet/guides/nui/nui-components/CheckBox.md]: Thank you for your suggestion.
I have updated the patch as you suggested.

File: docs/application/dotnet/guides/nui/nui-components/RadioButton.md (hunk 3 above)
Comment (jaehyun0cho) [reply docs/application/dotnet/guides/nui/nui-components/RadioButton.md]: Thank you for your suggestion.
I have updated the patch as you suggested.

**Review by mijin85cho — APPROVED** [review]
Thanks for the updates.

**Pull request merged**